src/main/java/com/iexec/core/replicate/ReplicatesController.java

```diff
@@ -103,6 +104,18 @@ public ReplicatesController(ReplicatesService replicatesService,
         statusUpdate.setDate(new Date());
         statusUpdate.setSuccess(ReplicateStatus.isSuccess(statusUpdate.getStatus()));
 
+        final var replicateStatusUpdateError = replicatesService.canUpdateReplicateStatus(
```

Comment:
This is just my opinion, I find it more readable when the explicit type is used.

Reply:
I was unsure about this one :laughing: 
I'm not fond of `var` but I think they can make reading easier when correctly used: when type and variable name are the same and long or when there is a `new ExplicitClassName()`. If that's much harder to read for you, I could turn it back to `ReplicateStatusUpdateError`. Just tell me :)

Reply:
In this specific case, the type of `replicateStatusUpdateError` is not straightforward yes.